src/server/package/src/model_explorer/pytorch_exported_program_adater_impl.py

```diff
@@ -30,10 +31,18 @@
 
 class PytorchExportedProgramAdapterImpl:
 
-  def __init__(self, ep: torch.export.ExportedProgram):
+  def __init__(
+      self, ep: torch.export.ExportedProgram, settings: Union[Dict, None]
+  ):
     self.ep = ep
     self.gm = self.ep.graph_module
     self.inputs_map = self.get_inputs_map()
+    if settings is None:
+      # Assigns the default values
+      settings = {
+          'const_element_count_limit': 16,
+      }
+    self.settings = settings
```

Comment:
Probably need 

```python
self.settings.setdefault('const_element_count_limit', 16)
```
?

Reply:
When `settings` is `None`, we will need to initialize a new `Dict` with default values for `settings`.